js/views/socketVw.js

```diff
@@ -26,28 +27,12 @@ module.exports = Backbone.View.extend({
       };
       this.socketConnection.onclose = this.socketClose();
     } catch(exception){
-      try{
-        console.log(socketAddress, window.polyglot.t('errorMessages.socketError') + "<br/><br/>" + exception);
-        //use default socket
-        this.socketConnection = new WebSocket('ws://localhost:18466');
-        this.socketConnection.onopen = this.socketOpen();
-        this.socketConnection.onmessage = function (e) {
-          self.socketMessage(e);
-        };
-        this.socketConnection.onerror = function (e) {
-          self.socketError(e);
-        };
-        this.socketConnection.onclose = this.socketClose();
-      }catch(exception){
-        console.log(exception);
-        //create a fake object so the rest of the view doesn't error out
-        this.socketConnection = {readyState: 0};
-        $('.js-loadingMessageModal').removeClass('hide');
-        $('.js-indexLoadingMsg1').text("WebSockets Cannot be reached.");
-        $('.js-indexLoadingMsg2').text("Stored URL of " + socketAddress + "has failed. Default address of ws://localhost:18466 has also failed.");
-        $('.js-indexLoadingMsg3').text("Interface will continue loading, but some functionality will not be available.");
-      }
+      console.log(socketAddress, window.polyglot.t('errorMessages.socketError') + "<br/><br/>" + exception);
+      messageModal.show('WebSockets cannot be reached.', '<i>Interface will continue loading, but some functionality will not be available.</i>');
```

Comment:
Currently, the Server Connect modal doesn't take into account this socket to determine connectivity. It basically, factors in the Heartbeat socket + the rest api.

For now, I mostly left this view in tact and it detects itself whether a connection failed displaying a simple message if so. The problem is that there's no way from this message to change the port (e.g. if you saved in the wrong port). In that case you'd need to kill your server to get the Server Connect modal to pop up so you could adjust the port.

I think a quick fix would be to add a button in the dialog that kicks off the Server Connect modal. Eventually, we could refactor this guy and work it into the Server Connect modal's connection algorithm.